pkg/gui/controllers/helpers/branches_helper.go

```diff
@@ -23,37 +25,49 @@ func NewBranchesHelper(c *HelperCommon, worktreeHelper *WorktreeHelper) *Branche
 	}
 }
 
-func (self *BranchesHelper) ConfirmLocalDelete(branch *models.Branch) error {
-	if self.checkedOutByOtherWorktree(branch) {
-		return self.promptWorktreeBranchDelete(branch)
+func (self *BranchesHelper) ConfirmLocalDelete(branches []*models.Branch) error {
+	if len(branches) > 1 {
+		if lo.SomeBy(branches, func(branch *models.Branch) bool { return self.checkedOutByOtherWorktree(branch) }) {
+			return errors.New(self.c.Tr.SomeBranchesCheckedOutByWorktreeError)
+		}
```

Comment:
are we completely disallowing branch deletion if it's checked out by another worktree? I noticed the previous implementation prompted the user first rather than outright rejecting it.

Reply:
If one of the selected branches is checked out in another worktree, yes. I did this because I found it too complex to wrap my head around what should happen in that case (for a range selection).

If you select those branches one by one, you still get the prompt for what to do with the worktree as before.